### PR TITLE
[13.0][MIG] hr_employee_document

### PR DESCRIPTION
**Pull request by newtratip**
google_drive_link field is not in employee form, I removed it.

This PR is standard migration. Ready for review.

### CODE REVIEW

File: hr_employee_document/__manifest__.py
Comment (astirpe) [inline hr_employee_document/__manifest__.py]: Dependency with `attachment_indexation` is not necessary: with this module the attachments are already accessible by smart button.

**Review by astirpe — CHANGES_REQUESTED** [review]
Works fine!

File: hr_employee_document/__manifest__.py
Comment (alexey-pelykh) [inline hr_employee_document/__manifest__.py:5]: Shouldn't this auto-format change be a separate commit?

File: hr_employee_document/views/hr_employee_views.xml
Comment (alexey-pelykh) [inline hr_employee_document/views/hr_employee_views.xml]: This is part of migration or improvement?

Comment (astirpe) [reply hr_employee_document/views/hr_employee_views.xml]: Part of migration

Comment (newtratip) [reply hr_employee_document/views/hr_employee_views.xml]: Yes, Migration

**Comment by alexey-pelykh** [comment]
Please tidy up the commit history, merge commit shouldn't even be there

File: hr_employee_document/views/hr_employee_views.xml
Comment (alexey-pelykh) [reply hr_employee_document/views/hr_employee_views.xml]: That field is gone from Odoo?

**Comment by alexey-pelykh** [comment]
Please also take a look at #799 as well

**Comment by alexey-pelykh** [comment]
And cherry-pick #802

**Comment by alexey-pelykh** [comment]
@Saran440 I think those cherry-picks should go before the migration and formatting commits

**Comment by Saran440** [comment]
@alexey-pelykh Okay, We will fix it. Thank you for your advise.
@newtratip Would you like to do this?

**Comment by rafaelbn** [comment]
Hello @newtratip are you going to conitnue with this PR? Thank you!

File: hr_employee_document/__manifest__.py
Comment (newtratip) [reply hr_employee_document/__manifest__.py]: Edited.

File: hr_employee_document/views/hr_employee_views.xml
Comment (newtratip) [reply hr_employee_document/views/hr_employee_views.xml]: @alexey-pelykh I can not see `google_drive_link` field in employee view.
Selection_459

**Comment by newtratip** [comment]
> And cherry-pick #802

> @Saran440 I think those cherry-picks should go before the migration and formatting commits

@alexey-pelykh Done

**Comment by alexey-pelykh** [comment]
Please follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch in splitting formatting and actual migration

**Comment by newtratip** [comment]
@alexey-pelykh done, please check again.

**Review by alexey-pelykh — APPROVED** [review]

**Comment by newtratip** [comment]
@rafaelbn Yes, I continue with this PR. Can you help review please ?

File: hr_employee_document/i18n/fr.po
Comment (alex-guardado) [inline hr_employee_document/i18n/fr.po:7]: I don't know if this could cause a translation error. Maybe new translations should be made or the file should be changed to a more updated one. As I said i don't know if it's an error or it doesn't matter. :)

Comment (newtratip) [reply hr_employee_document/i18n/fr.po:7]: I don't sure translation file will be updated by oca-travis ? This below module just merged and oca-travis update automatically.
Selection_480

Do you know about this ? @kittiu @Saran440

Comment (Saran440) [reply hr_employee_document/i18n/fr.po:7]: @alex-guardado I think it doesn't matter. module will translate from .pot and bot will auto generate after merged.

Comment (alex-guardado) [reply hr_employee_document/i18n/fr.po:7]: Okey, thank you for your answers. I didn't know how it worked exactly. Thank you :)

**Review by astirpe — APPROVED** [review]
Code review only. LGTM.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Pull request merged**
